core/src/test/java/com/linecorp/armeria/common/DefaultResponseHeadersBuilderTest.java

```diff
@@ -101,6 +101,25 @@ void validation() {
                 .hasMessageContaining(":status");
     }
 
+    @Test
+    void testSetCookieBuilder() {
+        final Cookie cookie = Cookie.of("cookie", "value");
+        final ResponseHeaders headers = ResponseHeaders.builder(HttpStatus.OK)
+                .setCookie(cookie)
```

Comment:
Could you check the indentation?

Reply:
I reformat as below.
```java
final RequestHeaders headers = RequestHeaders
                .builder()
                .path("/")
                .method(HttpMethod.GET)
                .cookie(cookie)
                .build();
```
Like `testAcceptLanguageNoMatchingLanguages` in same `DefaultRequestHeadersBuilderTest.java`.

But in `DefaultHttpHeadersBuilderTest.java`, some code has a different style.
```java
@Test
    void testContentDisposition() {
        final HttpHeaders headers = HttpHeaders.builder()
                                               .set("Content-Disposition", "form-data; name=foo")
                                               .build();
```
Which one is preferred style in armeria?